coriolis/db/sqlalchemy/models.py

```diff
@@ -84,6 +84,7 @@ class TasksExecution(BASE, models.TimestampMixin, models.ModelBase,
                              backref=orm.backref('execution'))
     status = sqlalchemy.Column(sqlalchemy.String(100), nullable=False)
     number = sqlalchemy.Column(sqlalchemy.Integer, nullable=False)
+    type = sqlalchemy.Column(sqlalchemy.String(20))
```

Comment:
You need to add a transition script in `db/sqlalchemy/migrate_repo/versions` for any updates to the DB.